RequestAPI/settings.cfg

```diff
@@ -0,0 +1,8 @@
+[default]
```

Comment:
we've pushed the credentials. We can't erase this from commit history, so let's update the credentials.

Reply:
I thought I added it in the gitignore to not upload this file. I will update the creds and gitignore as well